tests/phpunit/admin/class-wc-rest-stripe-agentic-commerce-controller-test.php

```
@@ -0,0 +1,566 @@
+
+	/**
+	 * Unauthenticated GET requests should be refused.
+	 */
+	public function test_get_status_requires_auth(): void {
+		wp_set_current_user( 0 );
+
+		$request  = new WP_REST_Request( 'GET', self::STATUS_ROUTE );
+		$response = rest_do_request( $request );
+
+		$this->assertEquals( 401, $response->get_status() );
+	}
```

PHPMD (2.15.0)

[error] The method test_get_status_requires_auth is not named in camelCase. (undefined)

(CamelCaseMethodName)